bin/check-package-version

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env node
```

Comment:
I can't tell what does this script do, based on its name. Do you think we can choose a name that communicates its input/output better?